app/data/subjectSearchData2.json

```diff
@@ -1,931 +1,931 @@
 [
-    { "name": "Abattoir Worker", "qualType": "End-Point Assessment", "level": "Level 2" },
-    { "name": "Academic Professional", "qualType": "End-Point Assessment", "level": "Level 7" },
-    { "name": "Accident repair technician", "qualType": "End-Point Assessment", "level": "Level 3" },
-    { "name": "Accountancy or taxation professional", "qualType": "End-Point Assessment", "level": "Level 7" },
-    { "name": "Accounting", "qualType": "GCE AS/A Level", "level": "Level 3" },
+    { "name": "Abattoir Worker", "qualType": "Apprenticeship", "level": "Level 2" },
+    { "name": "Academic Professional", "qualType": "Apprenticeship", "level": "Level 7" },
+    { "name": "Accident repair technician", "qualType": "Apprenticeship", "level": "Level 3" },
+    { "name": "Accountancy or taxation professional", "qualType": "Apprenticeship", "level": "Level 7" },
+    { "name": "Accounting", "qualType": "GCE AS/A level", "level": "Level 3" },
     { "name": "Accounting", "qualType": "T Level", "level": "Level 3" },
```

Comment:
Does the 'T Level' need to be changed to lowercase too?

Reply:
Yes it does!!